pkg/build/api/v1/types.go

```diff
@@ -402,6 +408,17 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty"`
 }
 
+// JenkinsPipelineBuildStrategy holds parameters specific to a Jenkins Pipeline build.
+type JenkinsPipelineBuildStrategy struct {
```

Comment:
I.e. there is no "generic jenkins pipeline" strategy that *isn't* jenkinsfile based.